src/components/Layout.tsx

```diff
@@ -0,0 +1,8 @@
+interface AppLayoutProps {
+  children: React.ReactNode;
+}
+
+function Layout({ children }: AppLayoutProps) {
```

Comment:
```suggestion
interface Props {
  children: React.ReactNode;
}

const MainLayout = ({ children }: Props) => {
```

- const 키워드를 사용하는건 어떤가요?
- Layout이 추가될 여지가 있어서 MainLayout이라고 정확하게 명명해주는게 좋을것 같아요
- Props 이름은 `컴포넌트이름 + Props`로 하든지, 컴포넌트 이름을 생략하고 `Props` 로만 하는게 좋을 것 같아요

Reply:
[5a07e68](https://github.com/sjoleee/bemyidol-frontend/pull/12/commits/5a07e680090d7622493c68e9516a23a0ea1e2675)에 반영했습니다